.github/workflows/deploy-gcp.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       VERSION_FAMILY: ${{ github.event.client_payload.version_family }}
       SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
       SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      APPS_MARKETPLACE_API_URI: "https://apps.saleor.io/api/v2/saleor-apps"
```

Comment:
@andrzejewsky @magul looks like AppStore was not enabled in GCP. Please confirm it was by mistake, not intentional.

Reply:
I don't remember if Marketplace/AppStore was enabled at GCP, but don't believe (if it wasn't enabled) that there was a meaningful reason for that.

Be also aware, that:
 * currently we only deploy 3.5 there, we will move to 3.9 (and later to multiversion setup) - don't know when exactly that will happen; question to ask here - is AppStore supporting 3.5?
 * we plan to simplify our deployment pipelines (so there will be no separate GCP deployment, it will be just part of 3.5/3.9 standard deployment).